### PR TITLE
FAT potat :>

### PR DESCRIPTION
**Pull request by sylv256**

### CODE REVIEW

File: src/main/java/io/github/protonmc/proton/mixin/client/entities/LivingEntityRendererMixin.java [hunk 1]
```diff
@@ -26,14 +26,14 @@
...
+        super(dispatcher);
     }
-    
+
     @Unique
     private float creeperColor = 1.0F;
 
     @Inject(method = "render", at = @At("HEAD"))
-	public void getCreeperColor(T livingEntity, float f, float g, MatrixStack matrixStack, VertexConsumerProvider vertexConsumerProvider, int i, CallbackInfo ci) {
+    public void getCreeperColor(T livingEntity, float f, float g, MatrixStack matrixStack, VertexConsumerProvider vertexConsumerProvider, int i, CallbackInfo ci) {
```
Comment (namebee) [inline src/main/java/io/github/protonmc/proton/mixin/client/entities/LivingEntityRendererMixin.java:36]: Huh?

File: src/main/java/io/github/protonmc/proton/module/decoration/tater/TaterModule.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import io.github.protonmc.proton.module.ProtonModule;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
+
+// fixme: broken
+/**
+ * @author TehcJS
+ * @author mounderfod
+ */
+public class TaterModule extends ProtonModule {
```
Comment (namebee) [inline src/main/java/io/github/protonmc/proton/module/decoration/tater/TaterModule.java:14]: This should be disabled by default.

**Review by namebee — CHANGES_REQUESTED** [review]
Disable the tater block by default.

File: src/main/java/io/github/protonmc/proton/mixin/client/entities/LivingEntityRendererMixin.java (hunk 1 above)
Comment (sylv256) [reply src/main/java/io/github/protonmc/proton/mixin/client/entities/LivingEntityRendererMixin.java:36]: this was an older commit. i forgot to remove it

File: src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package io.github.protonmc.proton.module.decoration.tater;
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java:1]: How would @dzwdz 's category detection system handle this?

Comment (namebee) [reply src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java:1]: Absolutely fine.

Comment (YTG1234) [reply src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java:1]: There is no category named `TATER`

File: src/main/java/io/github/protonmc/proton/module/decoration/tater/TaterModule.java (hunk 2 above)
Comment (mounderfod) [reply src/main/java/io/github/protonmc/proton/module/decoration/tater/TaterModule.java:14]: :irritat:

File: src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java (hunk 3 above)
Comment (namebee) [reply src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java:1]: The category is called decoration though..

Comment (mounderfod) [reply src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java:1]: @TehcJS this needs to be `module.building` probably

**Review by mounderfod — CHANGES_REQUESTED** [review]

**Comment by namebee** [comment]
Merging for now, The disabled can be fixed once @dzwdz fixes tiny_config.

File: src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java (hunk 3 above)
Comment (YTG1234) [reply src/main/java/io/github/protonmc/proton/module/decoration/tater/FatPotatoBlock.java:1]: > The category is called decoration though..

I checked and the category detection checks for the last package part, which is `tater`.

**Comment by sylv256** [comment]
stealing this for something else, pls tell me if the license doesn't permit doing so

**Pull request merged**